### PR TITLE
Add flow-analysis feature flag to avoid AD0001 in pre-3.0 Roslyn

### PR DESCRIPTION
**Pull request by genlu**

### CODE REVIEW

File: src/GenerateAnalyzerRulesets/Program.cs [hunk 1]
```diff
@@ -268,13 +268,22 @@ void createPropsFile()
...
                 var fileWithPath = Path.Combine(directory.FullName, propsFileName);
                 File.WriteAllText(fileWithPath, fileContents);
             }
 
+            string getFlowAnalysisFeatureFlag()
+            {
+                return @"
+
+  <PropertyGroup>
+    <Features>$(Features);flow-analysis</Features>
```
Comment (sharwell) [inline src/GenerateAnalyzerRulesets/Program.cs]:
```suggestion
    <Features>$(Features), flow-analysis</Features> 
```

Comment (genlu) [reply src/GenerateAnalyzerRulesets/Program.cs]: Hmm, this is how we use it in this repo
https://github.com/dotnet/roslyn-analyzers/blob/5ed057905c4e4def7f9e5abd4baccf8fc7286b4a/src/Directory.Build.props#L10

Comment (sharwell) [reply src/GenerateAnalyzerRulesets/Program.cs]: https://github.com/dotnet/roslyn/blob/4ee95ab9ea1e5b6625912e284af342cffe7cf24e/eng/targets/Settings.props#L23

Comment (genlu) [reply src/GenerateAnalyzerRulesets/Program.cs]: OK, it seems either way is fine

http://source.roslyn.io/#Microsoft.Build.Tasks.CodeAnalysis/CompilerOptionParseUtilities.cs,21

Comment (genlu) [reply src/GenerateAnalyzerRulesets/Program.cs]: Also, why is IOperation feature flag is still used in Roslyn? Should remove that

Comment (genlu) [reply src/GenerateAnalyzerRulesets/Program.cs]: I will leave it as-is here. Do you have any other comments?

**Pull request merged**